# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/reftests/border-image/reftest.list
# Commit: 1f86740dbf1a
# Full Hash: 1f86740dbf1a19596fd07fbaf3b1100383f76876
# Author: Matt Woodrow <mwoodrow@mozilla.com>
# Date: 2021-05-19 08:32:22
# Regressor Bug: 1540737
# File Overlap Count: 1
# Description:
#   Bug 1540737 - Implement Paint() for all display items, and nsDisplayList so that we can paint without using FrameLayerBuilder. r=miko
#   
#   This creates a new leaf nsDisplayWrapper item for consumers that just want an empty wrapper, with a Paint method that asserts. This lets us leaves Paint pure-virtual on nsDisplayWrapList, and lets the compiler force us to implement it for the other leaf types.
#   
#   It also disables flattening of transform/opacity items. This is because they flatten the items even when used with FlattenedDisplayItemIterator, even though this doesn't add markers to replace them (only FLB does this). We could in theory fix this, but I don't think FLB performance matters much any more.
# ==============================================================================

diff -r fdee5a641aee -r 1f86740dbf1a layout/reftests/border-image/reftest.list
--- a/layout/reftests/border-image/reftest.list	Wed May 19 01:11:32 2021 +0000
+++ b/layout/reftests/border-image/reftest.list	Wed May 19 01:11:32 2021 +0000
@@ -89,5 +89,5 @@
 == svg-as-border-image-1c.html svg-as-border-image-1-ref.html
 == svg-as-border-image-2.html svg-as-border-image-2-ref.html
 == svg-as-border-image-3.html svg-as-border-image-3-ref.html
-fails-if(useDrawSnapshot) == svg-as-border-image-4a.html svg-as-border-image-4-ref.html
+fuzzy-if(useDrawSnapshot,16-16,80-80) == svg-as-border-image-4a.html svg-as-border-image-4-ref.html
 == svg-as-border-image-4b.html svg-as-border-image-4-ref.html